.circleci/config.yml

```diff
@@ -102,3 +111,10 @@ workflows:
                 - main
           requires:
             - release-management/release-package
+      - promote-channels:
+          filters:
+            branches:
+              only:
+                - main
+          requires:
+            - release-management/release-package
```

Comment:
So this will only be able to release older published archives, right?

For example, if commit 7.88.3 in package.json which has sha e0685df. Then commit that in the channels/beta. Then push that up to github. Circleci will run, release the package, then the pack-and-upload will run at the same time promote-channels will run. Promote channels will fail bacause the version hasn't been uploaded yet. 

If I push up the package.json change first. Wait for circleci to finish. Then checkin and push the channel update, the release-package won't run because it would have already been released.